hackathon_site/dashboard/frontend/src/components/inventory/Item/Item.tsx

```diff
@@ -12,8 +14,11 @@ interface ItemProps {
 }
 
 const Item = ({ image, title, total, currentStock }: ItemProps): ReactElement => {
+    const userType = useSelector(userTypeSelector);
     const stock = !currentStock
         ? "OUT OF STOCK"
+        : userType === "participant"
+        ? currentStock + " IN STOCK"
```

Comment:
<img width="369" alt="CleanShot 2022-12-23 at 16 13 56@2x" src="https://user-images.githubusercontent.com/36619969/209406065-ff5851e2-6629-4e27-aa2c-915000b17d56.png">

The API will still return the maximum number of items in stock. Do we want to completely prevent participants from seeing that? Or it is OK we just leave like this

Reply:
I think keeping the API response like that is fine because we need the data to show on the admin side. It's also not a big deal if they want to know the total number of items available, omitting it on the front-end is a UX choice.

Reply:
Sounds good!